### DISCUSSION TITLE
[Bug]: gcc 12.2.0 does not compile anything

### DISCUSSION THREAD

**Discussion by jeffrson** [ScoopInstaller/Main#4677] [Q&A]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

gcc

### Expected/Current Behaviour

I installed gcc: `scoop install gcc`.

Trying to compile even the simplest C source does not do anything - even if there's a compiler error. cpp, gcc, g++ seem to return immediately.

`gcc --version` works however.

### Steps to Reproduce

```console
scoop install gcc
gcc --version
=> gcc.exe (Rev6, Built by MSYS2 project) 12.2.0

get-content t.c

#include <stdio.h>

int main() {
  printf("Hello world!");
  return 0;
}
```

gcc -o t.exe t.c
.\t.exe
=>
> .\t.exe: The term '.\t.exe' is not recognized as a name of a cmdlet, function, script file, or executable program.
```


### Possible Solution

No idea

### Scoop and Buckets Version

```console
PowerShell version: 7.3.3
Useragent: Shovel/1.0 (+https://shovel.ash258.com) PowerShell/7.3 (Windows NT 10.0.19045.0; Win64; x64;)
Current Scoop (Shovel) version:
52fe1646 (HEAD -> main, origin/main, origin/HEAD) fix(Diagnostic): Better handling of migrated repos

'Base' bucket:
020acd5 (HEAD -> main, origin/main, origin/HEAD) starship: Update to version 1.14.1

'extras' bucket:
350eac6ba (HEAD -> master, origin/master, origin/HEAD) geogebra: Update to version 6.0.772.0

'java' bucket:
aed7f08f (HEAD -> master, origin/master, origin/HEAD) temurin8-nightly-jre: Update to version 8.0.372-6.0.202304101933

'main' bucket:
5f5767397 (HEAD -> master, origin/master, origin/HEAD) starship: Update to version 1.14.1
```

### Scoop Config

```console
{
  "lastupdate": "258|2023-04-11 10:56:03",
  "SCOOP_REPO": "https://github.com/Ash258/Scoop-Core",
  "SCOOP_BRANCH": "main",
  "MSIEXTRACT_USE_LESSMSI": true,
  "aria2-enabled": false
}
```

### PowerShell Version

```console
PSVersion                      7.3.3
PSEdition                      Core
GitCommitId                    7.3.3
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by rashil2000** [comment]
- The version of GCC in this bucket is 11.2.0, not 12.2.0.
- This is the output after doing `scoop install gcc`, indicating that it is working:

![image](https://user-images.githubusercontent.com/46838874/232459584-6361e30f-b65c-4c48-b7d0-648dfdbd5dd7.png)
